sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -64,11 +63,11 @@ typedef struct
   az_span user_agent;
 
   /**
-   * The `method_twin_content_type` is sent in the username to inform IoT Hub what format the device
-   * expects Twin Document and Direct Method payloads to use. If this option is not set, the default
+   * The `twin_content_type` determines the format to send in the username, which will inform IoT
+   * Hub what format the device expects Twin Document to use. If this option is not set, the default
    * is JSON.
    */
-  az_span method_twin_content_type;
+  az_iot_hub_client_option_twin_content_type twin_content_type;
```

Comment:
new fields should be at the end of the struct.

Reply:
why?  Dane had similar question. Not clear to me yet.

Reply:
Usually it's memory ordering. If you always add to the end, the position of fields never changes between versions (only amended). In theory there is a possibility of your code pulling in new headers but having old libs and the ordering would be off if you dot reference the fields.

Reply:
I guess I should say when I say "usually" I mean in my head. It could just be convention for other reasons